src/types.ts

```diff
@@ -48,6 +48,7 @@ export interface V3Keystore {
 
 export type AinOptions = {
   rawResultMode?: boolean;
+  axiosConfig?: any;
```

Comment:
How about using `AxiosRequestConfig` in axios library?

Reply:
Fixed